tests/resource_test.js

```diff
@@ -0,0 +1,65 @@
+/// <reference path="./steps.d.ts" />
+Feature('Resource test');
+
+var FirstLogin = false;
+var loginCookies = [];
+const pdfName = `Dummy PDF ${Math.random() * 1000}`;
+const collectionName = `Dummy Collection ${Math.random() * 1000}`;
+
+Before((I) => {
+    I.login('admin', 'password');
+});
+
+Scenario('test upload a PDF file and verify the upload process', (I) => {
+    I.seeInCurrentUrl('#dashboard');
+    I.click('Library'); I.wait(3);
+    I.seeInCurrentUrl('#resources');
+
+    I.click('Add New Resource'); I.wait(3);
```

Comment:
@waryhermit unfortunately, when I try the test sometimes wait is necessary for the test to be able to done, but I will try then.